### DISCUSSION TITLE
Bare metal address limitation

### DISCUSSION THREAD

**Discussion by XobrekX** [gotenberg/gotenberg#740] [Q&A] [answered]
Hey there, 

I just managed to write an ansible role setting up gotenberg on bare metal. 
I have just one thing left, is there an option to limit the exposure address to localhost/127.0.0.1? 
I just found the option for the API port. 

Thanks in advance!

**Comment by gulien** [comment, marked as answer]
Hello @XobrekX,

Not such option, as with the default Docker context it does not make sense ^^

**Reply by XobrekX** [reply]
Darn, thanks anyway!
